scripts/cfFindIP.sh

```diff
@@ -168,18 +168,53 @@ function fncCheckSubnet {
 				then
 					ipConfigFile="$configDir/config.json.$ip"
 					cp "$scriptDir"/config.json.temp "$ipConfigFile"
-					sed -i "s/IP.IP.IP.IP/$ip/g" "$ipConfigFile"
+					if [[ "$osVersion" == "Mac" ]]
```

Comment:
You should do all the checks with one if
```shell
if mac
then
1
2
3
4
elif Linux
then
1
2
3
4
fi
```

Reply:
done.